presto-main/src/main/java/com/facebook/presto/execution/ScheduledSplit.java

```diff
@@ -60,7 +61,7 @@ public Split getSplit()
     @Override
     public int hashCode()
     {
-        return Longs.hashCode(sequenceId);
+        return Objects.hash(planNodeId, sequenceId);
```

Comment:
You need to update the `equals` then. Why are we changing that? How the `equals` / `hashcode` is used for this object? Why don't we include the `split` to the comparison?